src/main/java/org/examplesolid/application/util/Constants.java

```diff
@@ -0,0 +1,5 @@
+package org.examplesolid.application.util;
```

Comment:
Sera adecuado dejar esos utilities en esta capa?

Reply:
En que capa crees tu que deberia ir ? yo la deje en la capa de aplicación porque me parecia que es un caso de uso y no hacia parte del dominio

Reply:
En lo que tu tienes, esas constantes deben ir en la capa de `Infrastructure` ya que esa capa es la responsable the proveer detalles tecnicos de la applicacion como el accesso a bases de datos, apis, lectura de archivos, etc.